src/styles/primeng-theme/components/_multiselect.scss

```diff
@@ -0,0 +1,158 @@
+#multiselect-item {
+    margin: 0;
+    padding: $basic-padding;
+
+    &:hover {
+        @extend #ui-hover-state;
+    }
+}
+
+.ui-multiselect {
+    @extend #inputtext-normal-border;
+    border-radius: 0;
+
+    &.ui-state-focus {
+        @extend #inputtext-active-border;
+    }
+
+    &:not(.ui-state-disabled) {
+        &:hover,
+        &:active {
+            @extend #field-hover-state;
+        }
+
+        &.ui-state-focus {
+            .ui-multiselect-trigger {
+                border-color: $dropdown-active-color;
+            }
+        }
+    }
+
+    &.ui-state-disabled {
+        background: none;
+
+        .ui-multiselect-trigger {
+            color: $dropdown-disabled-color;
+        }
+
+        .ui-multiselect-label {
+            &:hover {
+                color: $dropdown-disabled-color;
+            }
+        }
+    }
+
+    .ui-multiselect-trigger {
+        align-items: center;
+        display: flex;
+        justify-content: center;
+        width: $dropdown-height;
+
+        .fa {
+            margin: 0;
+        }
+    }
+
+    .ui-multiselect-label {
+        border: 0;
+        line-height: $dropdown-height;
+        padding: 0 $dropdown-height + $basic-padding 0 0;
+        max-height: $dropdown-height;
+    }
+}
+
+p-multiselect {
+    outline-width: 0;
+    &.ng-dirty.ng-invalid.ng-touched,
+    &.ng-invalid.ng-touched {
+        .ui-multiselect:not(.ui-multiselect-open) {
+            .ui-inputtext {
+                @extend #inputtext-error-border;
+            }
+            .ui-multiselect-label,
+            .ui-inputtext,
+            .ui-multiselect-trigger {
+                color: $red;
+            }
+        }
+    }
+}
+
+.ui-multiselect-panel {
+    padding: $basic-padding * 2;
+    min-width: 300px;
+    .ui-widget-content {
+        .ui-state-highlight {
+            background: none;
+            color: $dropdown-active-color;
+        }
+    }
+    .ui-widget-header {
+        padding: 0;
+
+        .ui-chkbox {
+            position: absolute;
+            top: 10px;
+            z-index: 1;
+        }
+
+        .ui-multiselect-close {
+            display: none;
+        }
+
+        .ui-inputtext {
+            @extend #inputtext-normal-border;
```

Comment:
Shouldn't this being inherit from the original ```.ui-inputtext``` you sure you need all this?

Reply:
right, it was not working when the element is not attached to the body. "appendTo="body"". I left only what is needed now.